diff --git a/sub_mod_2/src/test/java/p_1/AssafTest.java b/sub_mod_2/src/test/java/p_1/AssafTest.java
index dbc8dbd..7cc8055 100644
--- a/sub_mod_2/src/test/java/p_1/AssafTest.java
+++ b/sub_mod_2/src/test/java/p_1/AssafTest.java
@@ -6,10 +6,19 @@ import static org.junit.jupiter.api.Assertions.*;
 
 class AssafTest {
 
+    Assaf a = new Assaf();
     @Test
     void moo() {
     }
     @Test
     void goo() {
     }
+    @Test
+    void compTest() {
+        assertEquals(a.moo(), 6);
+    }
+    @Test
+    void notCompTest() {
+        assertEquals(a.notComp(), 6);
+    }
 }
\ No newline at end of file
