diff --git a/sub_mod_1/src/test/java/p_1/AmitTest.java b/sub_mod_1/src/test/java/p_1/AmitTest.java
index 973b53b..c129374 100644
--- a/sub_mod_1/src/test/java/p_1/AmitTest.java
+++ b/sub_mod_1/src/test/java/p_1/AmitTest.java
@@ -17,6 +17,6 @@ class AmitTest {
 
     @Test
     void RTerrorTest() {
-        assertEquals(0,a.RTerror());
+        assertEquals(1,a.RTerror());
     }
 }
\ No newline at end of file
